content/departments/engineering/dev/career-development/framework.md

```diff
@@ -26,6 +26,12 @@ These are the expectations for ICs _after they have completed their onboarding_.
 We expect every engineer to eventually reach at least level 3.
 It is the responsibility of your manager to track this, and to ensure that you are given the support and opportunities needed for career growth.
 
+## Successful engineers at Sourcegraph
+
+In order to make the level descriptions below a bit more tangle, we also have the following document: [Successful engineers at Sourcegraph](./successful-engineers.md).
+
+The goal of that document is to drive home that point that the ladder is not a series of checkboxes to be checked-off sequentially. There are many different ways to have impact across multiple levels here.
```

Comment:
We're removing the "checkbox" language completely, but this is fine for now. 

Although FYI that you may have a merge conflict with Lauren.